Fixed #30382 -- Allowed specifying parent classes in force_insert of Model.save(). (#16830)
django/django: docs/releases/5.0.txt

Models
~~~~~~

> * The new ``create_defaults`` argument of :meth:`.QuerySet.update_or_create`.

Reviewer: Please revert.

  :ref:`Choices classes <field-choices-enum-types>` directly instead of
  requiring expansion with the ``choices`` attribute.

* :meth:`.Model.save` now passes the ``force_insert`` flag when saving models
  on inherited parents. This ensures that all parent models are saved according
  to the ``force_insert`` parameter.

* The new ``Model._meta.get_non_abstract_parent_list()`` method is a new
  addition to the model's meta options. It returns all the non-abstract parent
  classes of the model as a list.

> * ``Model.queryset.create()`` method now sets the ``force_insert`` parameter to

Reviewer: Please move this to the `Backwards incompatible changes in 5.0` -> `Miscellaneous` section.

  ``False`` when calling the :meth:`.Model.save` method if the model is
  inherited from a non abstract parent class. This allows the creation of

>   inherited models without violating constraints.

Reviewer: These are not breaking changes but bugfixes and we don't document bugfixes in "feature" releases. Please revert changes to release notes.

  :ref:`Choices classes <field-choices-enum-types>` directly instead of
  requiring expansion with the ``choices`` attribute.

* The :ref:`force_insert <ref-models-force-insert>` argument of

>   :meth:`.Model.save` now allows specifying a tuple of parent classes to force
>   an ``INSERT`` statement.

Reviewer 2: ```suggestion
  :meth:`.Model.save` now allows specifying a tuple of parent classes that must
  be forced to be inserted.
```
